core/src/main/java/org/fao/geonet/languages/IsoLanguagesMapper.java

```diff
@@ -89,6 +99,28 @@ public void reinit() {
     }
 
 
+    /**
+     * Verify that the init loaded some data, if so then set verifiedLoaded to true so that we don't need to check again.
+     * Otherwise re-run the init() to try loading the object again.
+     */
+    public void checkInit() {
+        if (!validInit) {
+            // _langRepo will be null if this is not a bean - in which case just mark it as true and return.
```

Comment:
Is this something that can happen?

Reply:
I have updated the comment to indicate that it only occurs in the unit tests.